From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Fri, 30 Dec 2022 18:46:02 +0100
Subject: docs: Use inventary from packages

Most of the intersphinx modules can be taken from packages.

There are two exceptions while writing.
For matplotlib is no *-doc package or aquivalent data in the matplotlib
package available.
pyscaffold isn't packaged yet.

Forwarded: Not-Needed
---
 docs/conf.py | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 2d3e846..185b03f 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -314,15 +314,15 @@ latex_documents = [
 # -- External mapping --------------------------------------------------------
 python_version = ".".join(map(str, sys.version_info[0:2]))
 intersphinx_mapping = {
-    "sphinx": ("https://www.sphinx-doc.org/en/master", None),
-    "python": ("https://docs.python.org/" + python_version, None),
-    "matplotlib": ("https://matplotlib.org", None),
-    "numpy": ("https://numpy.org/doc/stable", None),
-    "sklearn": ("https://scikit-learn.org/stable", None),
-    "pandas": ("https://pandas.pydata.org/pandas-docs/stable", None),
-    "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
-    "setuptools": ("https://setuptools.pypa.io/en/stable/", None),
-    "pyscaffold": ("https://pyscaffold.org/en/stable", None),
+    "sphinx": ("/usr/share/doc/sphinx-doc/html/", None),
+    "python": ("/usr/share/doc/python3-doc/html", None),
+#    "matplotlib": ("https://matplotlib.org", None),
+#    "numpy": ("https://numpy.org/doc/stable", None),
+    "sklearn": ("/usr/share/doc/python-sklearn-doc/html", None),
+    "pandas": ("/usr/share/doc/python-pandas-doc/html", None),
+    "scipy": ("/usr/share/doc/python-scipy-doc/html", None),
+    "setuptools": ("/usr/share/doc/python-setuptools-doc/html", None),
+#    "pyscaffold": ("https://pyscaffold.org/en/stable", None),
 }
 extlinks = {
     "issue": (f"{repository}/issues/%s", "issue #%s"),
